debian/bareos-database-common.config.in

```diff
@@ -81,22 +76,6 @@ if [ -r @scriptdir@/bareos-config-lib.sh ]; then
                         db_set bareos-database-common/pgsql/app-pass ""
                     fi
                     ;;
-                mysql)
```

Comment:
Why not remove the whole case..esac block from line 76?

Reply:
I don't know, you were the last one who touched that file :smiley: